modules/ROOT/pages/configure-client-management-openid-task.adoc

```diff
@@ -107,20 +107,6 @@ Once this is successfully configured, you can apply the OpenID Connect OAuth Tok
 [NOTE]
 For Okta, the Okta admin needs to assign the dynamically generated clients to a user or a group of users in order for them to receive access tokens by sending over the Client ID and Client Secret.
 
-== Grant types
-
-If you have configured an OpenID Connect provider for client management, Anypoint Platform supports the following OAuth grant types by default when registering an API client application in the API portal:
-
-- Implicit
-- Authorization
-- Refresh Token
-
-[NOTE]
-The Refresh token can be selected only if the authorization grant type is also selected.
-
-In addition to these scopes, if you configure the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint Platform autopopulates its UI with all grant types supported by the provider. This includes grant types such as client credentials, password, and so on. To verify the grant types your provider supports, check their discovery endpoint using `__$ISSUER__/.well-known/openid-configuration`, where `_$ISSUER_` is the issuer configured as the provider. 
```

Comment:
We should keep this section. What grant types can be requested when creating a client is surfaced by this discovery endpoint. If unreachable, the three above ones are put in as default.

I think the "supported" wording is problematic in particular, and we should flip the order (i.e. the metadata gets looked at first, and if that fails we fall back to the default OpenID Connect scopes), but not say "supported".

Reply:
Didn't we just add this in response to a customer question, too?

Reply:
Sorry guys -- I didn't get an email notification about your comment for some reason.  

I don't quite understand.  Based on the doc that we worked on Seba, I thought all of the interaction to acquire the access token was directly between the client and the IDP.  

What exactly does "Anypoint Platform supports" mean in this context?

Is this the text that should be the main focus of this section, "If you configure the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint Platform autopopulates its UI with all grant types supported by the provider"?

Reply:
Yes, this is not about acquiring the token, but rather declaring what grant types you want the client you're registering to support. "Anypoint Platform supports" is still the wrong terminology, but Anypoint Platform surfaces the grant types that the IDP supports at client creation time.